content/pages/policies.md

```diff
@@ -0,0 +1,72 @@
+---
+$title@: User Policies
```

Comment:
This seems to be copy/paste? Not for our site?

Reply:
Ah - good flag. Removed the dead pages.